influxql/ast_test.go

```diff
@@ -150,11 +150,11 @@ func TestSelectStatement_SetTimeRange(t *testing.T) {
 	}
 
 	s := stmt.(*influxql.SelectStatement)
-	min, max := influxql.TimeRange(s.Condition)
+	min, max, _ := influxql.TimeRange(s.Condition)
 	start := time.Now().Add(-20 * time.Hour).Round(time.Second).UTC()
 	end := time.Now().Add(10 * time.Hour).Round(time.Second).UTC()
 	s.SetTimeRange(start, end)
-	min, max = influxql.TimeRange(s.Condition)
+	min, max, _ = influxql.TimeRange(s.Condition)
```

Comment:
Can you add a `MustTimeRange()` helper in `influxql_test` so that we're checking the error and panicking instead of ignoring it?